README.md

```diff
@@ -1,71 +1,83 @@
-# Deep Learning Pipelines for Apache Spark
+Deep Learning Pipelines for Apache Spark
+=============================================
+[![Build Status](https://travis-ci.org/databricks/spark-deep-learning.svg?branch=master)](https://travis-ci.org/databricks/spark-deep-learning)
 
-Deep Learning Pipelines provides high-level APIs for scalable deep learning in Python. The
-library comes from Databricks and leverages Spark for its two strongest facets:
-1. In the spirit of Spark and Spark MLlib, it provides easy-to-use APIs that enable deep learning
-in very few lines of code.
-2. It uses Spark's powerful distributed engine to scale out deep learning on massive datasets.
+- [Content](#content)
+  - [Building and running unit tests](#building-and-running-unit-tests)
+  - [Spark version compatibility](#spark-version-compatibility)
+  - [Quick user guide](#quick-user-guide)
+    - [Working with images in Spark](#working-with-images-in-spark)
+    - [Transfer learning](#transfer-learning)
+    - [Applying deep learning models at scale](#applying-deep-learning-models-at-scale)
+    - [Deploying models as SQL functions](#deploying-models-as-sql-functions)
+  - [Releases:](#releases)
 
-Currently, TensorFlow and TensorFlow-backed Keras workflows are supported, with a focus on model
-application and transfer learning on image data at scale, with hyper-parameter tuning in the works.
-Furthermore, it provides tools for data scientists and machine learning experts to turn deep
-learning models into SQL functions that can be used by a much wider group of users. It does not
-perform single-model distributed training - this is an area of active research, and here we aim to
-provide the most practical solutions for the majority of deep learning use cases.
 
-For an overview of the library, see the Databrick [blog post](https://databricks.com/blog/2017/06/06/databricks-vision-simplify-large-scale-deep-learning.html?preview=true) introducing Deep Learning Pipelines.
-For the various use cases the package serves, see the [Quick user guide](#quick-user-guide) section below.
+<a id="content"></a>
```

Comment:
I was porting this from org-mode, but now I found out that Emacs has a pretty good markdown mode, so I switched to directly updating the .md file.